app/assets/config/govuk_publishing_components_manifest.js

```diff
@@ -1,3 +0,0 @@
-// JS and CSS bundles for the gem
```

Comment:
How come you changed the name of this file? I found having this was quite useful for adding it to apps so they had:

```
link govuk_publishing_components_manifest
```

but it'd be better still if it wasn't needed, does using manifest.js mean it gets pulled in automatically?
For what it's worth the Rails engine documentation suggests `<engine_name>_manifest.js`: https://guides.rubyonrails.org/engines.html#generating-an-engine